docs/docs/community/run-locally.md

```diff
@@ -67,6 +69,48 @@ However, if you want to test certain parts of Novu or run it in production mode,
     </div>
 </details>
 
+<details>
+    <summary>Worker</summary>
+    <div>
+      <ul>
+        <li><code>NODE_ENV</code> (default: local)<br />The environment of the app. Possible values are: dev, test, prod, ci, local</li>
+        <li><code>PORT</code><br />The port on which the Worker app should listen on</li>
+        <li><code>STORE_ENCRYPTION_KEY</code><br />The encryption key used to encrypt/decrypt provider credentials</li>
+        <li><code>MAX_NOVU_INTEGRATION_MAIL_REQUESTS</code><br />The number of free emails that can be sent with the Novu email provider</li>
+        <li><code>NOVU_EMAIL_INTEGRATION_API_KEY</code><br />The Novu email provider Sentry API key</li>
+        <li><code>STORAGE_SERVICE</code><br />The storage service name: AWS, GCS, or AZURE</li>
+        <li><code>S3_LOCAL_STACK</code><br />The LocalStack service URL</li>
+        <li><code>S3_BUCKET_NAME</code><br />The name of the S3 Bucket</li>
+        <li><code>S3_REGION</code><br />The AWS region of the S3 Bucket</li>
+        <li><code>GCS_BUCKET_NAME</code><br />The name of the GCS Bucket</li>
+        <li><code>AZURE_ACCOUNT_NAME</code><br />The name of the Azure account</li>
+        <li><code>AZURE_ACCOUNT_KEY</code><br />The Azure account key</li>
+        <li><code>AZURE_HOST_NAME</code><br />The Azure host name</li>
+        <li><code>AZURE_CONTAINER_NAME</code><br />The Azure container name</li>
+        <li><code>AWS_ACCESS_KEY_ID</code><br />The AWS access key</li>
+        <li><code>AWS_SECRET_ACCESS_KEY</code><br />The AWS secret access key</li>
+        <li><code>REDIS_HOST</code><br />The domain / IP of your redis instance</li>
+        <li><code>REDIS_PORT</code><br />The port of your redis instance</li>
+        <li><code>REDIS_PASSWORD</code><br />Optional password of your redis instance</li>
+        <li><code>REDIS_DB_INDEX</code><br />The Redis database index</li>
+        <li><code>REDIS_CACHE_SERVICE_HOST</code><br />The domain / IP of your redis instance for caching</li>
+        <li><code>REDIS_CACHE_SERVICE_PORT</code><br />The port of your redis instance for caching</li>
```

Comment:
@LetItRock we would need to do the last sync from next today after we will merge Pablo's PR for the cluster caching support and to update the variable here. You can connect with @p-fernandez regarding that, as we might need to move some the caching services to application generic.

Reply:
@scopsy all the caching services are already in the `application-generic` in my PR, but if @p-fernandez has started from a different branch then yes, there will be conflicts